CHANGELOG.md

```diff
@@ -1,3 +1,191 @@
+## Release 0.14.0
+
+Highlights:
+
+This release comes with two main new features.
```

Comment:
@peterbourgon There was a colon but @peterbourgon  suggested the opposite and I changed it (he rightfully claimed that it would get confused with a title).